src/openrct2/paint/Paint.cpp

```diff
@@ -537,7 +626,11 @@ constexpr std::array _paintArrangeFuncs = {
 void PaintSessionArrange(PaintSessionCore& session)
 {
     PROFILED_FUNCTION();
-    return _paintArrangeFuncs[session.CurrentRotation](session);
+    if (gPaintStableSort)
```

Comment:
If multithreading is enabled, this is a data race

Reply:
Hmm, actually no, this is not a data race. We set the variable in the main thread and add paint session tasks from the same main thread after it's been set. It would've been a data race if we set it in a different thread.